source/win-main/MainEditor.vue

```diff
@@ -123,25 +123,29 @@ const props = defineProps({
   distractionFree: {
     type: Boolean,
     required: true
+  },
+  file: {
+    type: Object as () => OpenDocument,
+    required: true
   }
 })
 
+// TODO: Update the activeFileDescriptor as soon as the appropriate event is
```

Comment:
Is this TODO something for the future or for this PR? If it is meant for this PR, we could add a todo item in the PR description to not forget this.

Reply:
That was mainly a TODO for myself, and I just checked – I finished that one already :D